config/initializers/4-accounts.rb

```diff
@@ -0,0 +1,6 @@
+accounts = ActiveSupport::OrderedOptions.new
+# configure how long a login token is valid for
+accounts.default_login_token_expiration_period = 2.days
```

Comment:
I'm not finding info on this - does it expire the logged in session after two days? or is this a temporary login token thing?

Reply:
ah, i see it was just a move from here https://github.com/openstax/accounts/pull/1260/files?diff=split&w=0#diff-c1fd91cb1911a0512578b99f657554526f3e1421decdb9e908712beab57e10f9L30

Reply:
yeah sorry, I moved the config in application.rb and some of environment.rb to individual initializers